goon-universe/app.js

```diff
@@ -152,6 +152,7 @@ router.route('/scenarios/initialReflection')
         scenarioID = req.body.scenarioID
         studentID = req.body.studentID
         data = req.body.data
+        let flag = true
```

Comment:
Could this be renamed to something more descriptive?

Reply:
Done